easybuild/tools/modules.py

```diff
@@ -831,33 +834,54 @@ def available(self, mod_name=None):
 
         return correct_real_mods
 
+    def compile_cache(self, cache_fp):
+        """Compile Lmod cache file using luac."""
+        if build_option('compile_lmod_caches'):
+            # FIXME: ask Lmod to compile, to ensure that luac that matches the lua used by Lmod is used?
```

Comment:
that's just a side-effect, but yes, that's good

it's important that Lmod uses the `luac` that matches the `lua` it's using, since the Lua version has to be part of the filename of the compiled cache (I still need to fix that here)